pkgs/by-name/at/attic-client/package.nix

```diff
@@ -11,13 +11,13 @@
 }:
 rustPlatform.buildRustPackage {
   pname = "attic";
-  version = "0.1.0";
+  version = "0-unstable-2024-08-01";
```

Comment:
Why the `0-` prefix?
Anyway in the upstream derivation the `version` is `0.1.0`, and also in all the `Cargo.toml`s it says `0.1.0`. Consequently also `attic --version` reports `0.1.0`.

[Here](https://github.com/NixOS/nixpkgs/pull/274481#issuecomment-1994062002) my message about it.

Said this, I was expecting a release soon when I did that change, so probably renaming to `unstable-<date>` makes more sense now.

Reply:
> Why the `0-` prefix?

https://github.com/NixOS/nixpkgs/blob/102b1152a6ea69f4be9d66bde3cb3141d5642b6c/pkgs/README.md#package-naming

Reply:
Oh I didn't know, thanks :)